migrations/2022-07-25-101609_multiple_collections/up.sql

```diff
@@ -0,0 +1,168 @@
+CREATE TABLE multiple_collections
+(
+    id         BIGSERIAL PRIMARY KEY,
+    created_at TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at TIMESTAMP,
+    user_id    BIGSERIAL references users (id) ON DELETE CASCADE,
+    notes      TEXT,
+    name       TEXT      NOT NULL
+);
+
+CREATE UNIQUE INDEX multiple_collection_unique_name_user_not_deleted 
+ON multiple_collections (user_id, name) where deleted_at is null;
+
+--This is the same as 'Collections' but without the uniqueness constrain on user_id , document_id
+CREATE TABLE collection_items
+(
+    id                     BIGSERIAL PRIMARY KEY,
+    created_at             TIMESTAMP NOT NULL DEFAULT now(),
+    updated_at             TIMESTAMP NOT NULL DEFAULT now(),
+    deleted_at             TIMESTAMP,
+    document_id            BIGSERIAL references documents (id),
+    notes                  TEXT,
+    custom_name            TEXT,
+    user_id                BIGSERIAL REFERENCES users (id) ON DELETE CASCADE,
+    multiple_collection_id BIGSERIAL references multiple_collections (id)
```

Comment:
Should we add `ON DELETE CASCADE`? I guess we should support actually deleting collections at some point.

Reply:
I think I will do this for now. Let's create a placeholder for hard deleting